DataTool/ToolLogic/List/ListGeneralUnlocks.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataTool.Flag;
+using STULib;
+using STULib.Types;
+using static DataTool.Helper.IO;
+using static DataTool.Program;
+using static DataTool.Helper.Logger;
+using static DataTool.Helper.STUHelper;
+using DataTool.Models;
+using OWLib;
+using STULib.Types.Generic;
+
+// ReSharper disable SuspiciousTypeConversion.Global
+
+namespace DataTool.ToolLogic.List {
+    [Tool("list-general-unlocks", Description = "List general unlocks", TrackTypes = new ushort[] { 0x54 }, CustomFlags = typeof(ListFlags))]
+    public class ListGeneralUnlocks : JSONTool, ITool {
+        public void IntegrateView(object sender) {
+            throw new NotImplementedException();
+        }
+
+        public void Parse(ICLIFlags toolFlags) {
+            var unlocks = GatherUnlocks();
+
+            if (toolFlags is ListFlags flags) {
+                if (flags.JSON) {
+                    ParseJSON(unlocks, flags);
+                    return;
+                }
+            }
+
+            foreach (KeyValuePair<string,  HashSet<ItemInfo>> groupPair in unlocks) {
+                if (groupPair.Value?.Count == 0) continue;
+
+                if (groupPair.Value != null) {
+                    Log(groupPair.Key);
+                    foreach (ItemInfo unlock in groupPair.Value) {
+                        if (unlock?.Name == null) continue;
+                        Log("\t{0} ({1} {2})", unlock.Name, unlock.Rarity, unlock.Type);
+                    }
+                }
+            }
+        }
+
+        public Dictionary<string, HashSet<ItemInfo>> GatherUnlocks() {
+            Dictionary<string, HashSet<ItemInfo>> @return = new Dictionary<string, HashSet<ItemInfo>>();
+            foreach (ulong key in TrackedFiles[0x54]) {
+                STUGlobalInventoryMaster invMaster = GetInstance<STUGlobalInventoryMaster>(key);
+                if (invMaster == null) continue;
+
+                @return["Achievement"] = GatherUnlocks(invMaster.AchievementUnlocks?.Unlocks?.Select(it => (ulong)it));
+
+                @return["Standard/Common"] = new HashSet<ItemInfo>();
```

Comment:
Small gripe, but we should use `Standard` instead of `Standard/Common` here now. 

Standard/Common doesn't make much sense, non common items are in there too